src/_css/main.css

```diff
@@ -224,6 +224,10 @@ hr {
   align-items: flex-end;
 }
 
+#footer nav {
+  font-size : 15px;
```

Comment:
I'd prefer not adding a new font-size.
How about removing the dots `.` separators between the nav links and add `margin-right: 0.5em` for `footer nav a`?

Reply:
I agree with your opinion.
I've just added the modified version. (removing dots and adding `margin-right` to them, and things i had to do to align them in the right position.)

The line-break still remains in the mobile version, but i think it's not a big things.

Reply:
adding a .prettierrc sounds good to me